keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -232,13 +245,13 @@ relative priority ordering of all configured rules and CNP instances.
 
 #### Rule identifiers
 
-In order to collect traffic statistics corresponding to
-a rule, it is necessary to identify the rule which allows/denies that traffic.
+In order to collect traffic statistics corresponding to a rule in the future,
```

Comment:
I think we should leave "traffic statistics" out of here here and just argue that with the addition of multiple "appliedTos" Being able to name rules would help keep things less complex